### PR TITLE
Update instructions to build sdk packages in Linux

### PR DESCRIPTION
**Pull request by poornas**
This PR updates the MAINTAINERS.md with instructions to build the Nuget packages for .net framework and core in Linux. This is a first step to creating a release script to automate this process (Refer: https://github.com/minioops/q/issues/11).

The build configurations have been reorganized and renamed as Release.Core and Release.Net452 to faciliate this, apart from updates to Nuspec files to change the binary paths.

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -49,10 +57,20 @@ or this project can be selected from Visual Studio and click Run.
 > nuget pack Minio.core.nuspec
 ... package built ...
 ```
+#### Upload the package to nuget.org
+Login to nuget.org(https://www.nuget.org) and find the API Key for Minio
+```powershell
```
Comment (harshavardhana) [inline MAINTAINERS.md]: follow commands don't look like are for `powershell` starting with `export` - perhaps we can state that these instructions are for linux and change the markdown styling to `sh` instead?

File: MAINTAINERS.md [hunk 2]
```diff
@@ -49,10 +57,20 @@ or this project can be selected from Visual Studio and click Run.
...
+>  export MINIO_API_KEY=??
+>  nuget setApiKey $MINIO_API_KEY
+>  nuget push .\Minio.1.0.2.nupkg -Apikey $MINIO_API_KEY -src https://nuget.org
+>  nuget push .\Minio.NetCore.1.0.2.nupkg -Apikey $MINIO_API_KEY -src https://nuget.org
+
+```
+#### Publish package to nuget.org
+Login to nuget.org to [publish](https://www.nuget.org/account/Packages)  the new package versions through browser.
```
Comment (harshavardhana) [inline MAINTAINERS.md]: Is this necessary can this be automated?

File: MAINTAINERS.md [hunk 3]
```diff
@@ -49,10 +57,20 @@ or this project can be selected from Visual Studio and click Run.
...
 
-#### Go to nuget.org
-Login to nuget.org to [upload a package through browser](https://www.nuget.org/users/account/LogOn?ReturnUrl=%2Fpackages%2Fupload).
-Upload both the packages generated by the previous step.
+- Select the new package versions and edit Release notes.
```
Comment (harshavardhana) [inline MAINTAINERS.md]: Can this be automated?

File: MAINTAINERS.md (hunk 1 above)
Comment (poornas) [reply MAINTAINERS.md]: ok, could you try doing a restore first? ``` msbuild /p:Configuration=Release.Net452 /t:Restore ```
I will update the instructions.

File: MAINTAINERS.md (hunk 2 above)
Comment (poornas) [reply MAINTAINERS.md]: yes, replaced this with a verification step.

File: MAINTAINERS.md (hunk 3 above)
Comment (poornas) [reply MAINTAINERS.md]: yes, updated PR.

File: .travis.yml [hunk 4]
```diff
@@ -9,6 +9,6 @@ branches:
 install:
 
 script:
-- dotnet restore /p:Configuration=.netcore 
-- dotnet publish  --output out /p:Configuration=.netcore
-- dotnet /home/travis/build/minio/minio-dotnet/Minio.Functional.Tests/bin/Debug/netcoreapp1.1/Minio.Functional.Tests.dll
+- dotnet restore /p:Configuration=Release.Core 
+- dotnet publish  --output out /p:Configuration=Release.Core
+- dotnet /home/travis/build/minio/minio-dotnet/Minio.Functional.Tests/bin/Release.Core/netcoreapp1.1/Minio.Functional.Tests.dll
```
Comment (harshavardhana) [inline .travis.yml]: ```dotnet Minio.Functional.Tests/bin/Release.Core/netcoreapp1.1/Minio.Functional.Tests.dll```

**Comment by poornas** [comment]
@krisis and @harshavardhana , this is waiting on your reviews.

**Comment by harshavardhana** [comment]
I have tested this other than publishing a release. LGTM.

**Pull request merged**